packages/core/src/components/menu/menuItem.tsx

```diff
@@ -177,6 +177,7 @@ export class MenuItem extends AbstractPureComponent2<MenuItemProps & React.Ancho
         const target = React.createElement(
             tagName,
             {
+                role: "menuitem",
```

Comment:
let's also add `"aria-expanded": hasSubmenu` here as suggested in the linked [docs](https://w3c.github.io/aria-practices/examples/menubar/menubar-navigation.html).

```suggestion
                "aria-expanded": hasSubmenu,
                role: "menuitem",
```

there's a separate issue where `aria-hassubmenu` is being added to the wrong element (the wrapping `<span>` of the popover target rather than the `<a role="menuitem">` element), but that's a problem I will fix later when I create a MenuItem2 variant which uses Popover2 instead of Popover:

<img width="539" alt="image" src="https://user-images.githubusercontent.com/723999/170504754-52cde185-d5b9-450a-a3e5-13fbcf12b219.png">

Reply:
oops, this is wrong, sorry, "expanded" state is different from simply _having_ a submenu. setting "aria-expanded" will not be this simple. I will revert my suggestion.